crates/cli-support/src/js/mod.rs

```diff
@@ -437,6 +437,18 @@ impl<'a> Context<'a> {
                         module = import.meta.url.replace(/\\.js$/, '_bg.wasm');
                     }"
             }
+            OutputMode::NoModules { .. } => {
+                "\
+                    if (typeof module === 'undefined') {
+                        let src
```

Comment:
Could this and the lines below also include semicolons? Other than that looks good to me!

Reply:
Done.